lms/djangoapps/lms_xblock/test/test_settings_service.py

```diff
@@ -0,0 +1,76 @@
+"""
+Tests for SettingsService
+"""
+
+import ddt
+import mock
+from unittest import TestCase
+
+from django.conf import settings
+from django.test.utils import override_settings
+
+from xblock.runtime import Mixologist
+from lms.djangoapps.lms_xblock.runtime import SettingsService
+
+
+class _Constants(object):
+    """ 'Static' class to hold constants used in `override_settings` decorators in TestSettingsService """
+    xblock_setting_key1 = 'dummy_block'
+    xblock_setting_key2 = 'other_dummy_block'
```

Comment:
You could also just move these static constants to be defined on `TestSettingsService` rather than in a separate class.